src/stdlib/compact.rs

```diff
@@ -394,57 +404,64 @@ mod test {
                         ])),
                     ),
                     (KeyString::from("key3"), Value::from(2)),
-                ]),
+                ])
+                .into(),
                 CompactOptions::default(),
             ),
             (
-                ObjectMap::from([
+                BTreeMap::from([
                     (KeyString::from("key1"), Value::from(1)),
                     (
                         KeyString::from("key2"),
                         Value::Object(ObjectMap::from([(KeyString::from("key1"), Value::Null)])),
                     ),
                     (KeyString::from("key3"), Value::from(2)),
-                ]),
-                ObjectMap::from([
+                ])
+                .into(),
+                BTreeMap::from([
                     (KeyString::from("key1"), Value::from(1)),
                     (
                         KeyString::from("key2"),
                         Value::Object(ObjectMap::from([(KeyString::from("key1"), Value::Null)])),
                     ),
                     (KeyString::from("key3"), Value::from(2)),
-                ]),
+                ])
+                .into(),
                 CompactOptions {
                     recursive: false,
                     ..CompactOptions::default()
                 },
             ),
             (
-                ObjectMap::from([
+                BTreeMap::from([
```

Comment:
I'm curious, with all the `impl From` that are provided for `ObjectMap`, can this conversion not stand as-is? Could it be handled with another `impl From`?